verifiers/internal/gha/provenance.go

```diff
@@ -297,7 +336,19 @@ func VerifyProvenance(env *dsselib.Envelope, provenanceOpts *options.ProvenanceO
 		if err := isValidDelegatorBuilderID(prov); err != nil {
 			return err
 		}
+
+		// If the --builder-id flag is empty and the builder used to build artifact is not from ExpectedBuilderPath
```

Comment:
let's not refer to CLI argument in comments, since this part of code is not aware of the CLI. Same for comment below.

Reply:
I agree, and I updated the scope of the comments!